### PR TITLE
feat: change from snake_case to camelCase

### PR DESCRIPTION
**Pull request by GeorgeNonis**
### Proposed changes

A description of the changes and why they are relevant.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed path definitions for each area in all renditions of the map, resulting in a ~30% decrease in the overall SVG size.
 - Changed the name of fields in the area types as follows:
   - `en_name` -> `asciiName`
-  - `display_name` -> `displayName`
-  - `en_term` -> `enTerm`
```
Comment (MartinP460) [inline CHANGELOG.md:28]: We don't need to change the name here since we are informing people of the change (:

File: MIGRATION.md [hunk 2]
```diff
@@ -16,8 +16,8 @@ Finally, all map SVG's are now ~30% smaller which should improve load times and
 2. **Viewbox**. All maps now have the same SVG viewbox dimensions of 1000 x 1215 so you should use these dimensions to customize the viewbox.
 3. **Renamed area fields**. Some keys in the area object have been renamed. These are specifically:
    - `en_name` -> `asciiName`
-   - `display_name` -> `displayName`
-   - `en_term` -> `enTerm`
```
Comment (MartinP460) [inline MIGRATION.md:20]: Same here

**Review by MartinP460 — CHANGES_REQUESTED** [review]
Looks good! I just left two small comments

File: CHANGELOG.md (hunk 1 above)
Comment (GeorgeNonis) [reply CHANGELOG.md:28]: Ctrl & Shift & F did the magic trick 😅

Comment (MartinP460) [reply CHANGELOG.md:28]: Caught in the cross-fire!

File: CHANGELOG.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed the name of fields in the area types as follows:
   - `en_name` -> `asciiName`
   - `display_name` -> `displayName`
-  - `en_term` -> `enTerm`
```
Comment (MartinP460) [inline CHANGELOG.md:28]: Also this one

File: MIGRATION.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ Finally, all map SVG's are now ~30% smaller which should improve load times and
 3. **Renamed area fields**. Some keys in the area object have been renamed. These are specifically:
    - `en_name` -> `asciiName`
    - `display_name` -> `displayName`
-   - `en_term` -> `enTerm`
```
Comment (MartinP460) [inline MIGRATION.md:20]: And this one (:

**Review by MartinP460 — CHANGES_REQUESTED** [review]

**Review by MartinP460 — APPROVED** [review]
Great work! 🥳

**Pull request merged**